php_memcached.c

```diff
@@ -231,14 +231,21 @@ zend_bool s_memc_valid_key_binary(zend_string *key)
 }
 
 static
-zend_bool s_memc_valid_key_ascii(zend_string *key)
+zend_bool s_memc_valid_key_ascii(zend_string *key, uint64_t verify_key)
```

Comment:
I was going to make this suggestion, but then I realized the setting is defined as 64-bits. So it goes! https://github.com/php-memcached-dev/php-memcached/blob/7fefcb792423906bba2bac66a49194f98bf11aa6/php_memcached.c#L4229

```suggestion
zend_bool s_memc_valid_key_ascii(zend_string *key, zend_bool verify_key)
```

Reply:
Yes, that is why I made it a uint64_t, but perhaps we should flip that one to a `REGISTER_MEMC_CLASS_CONST_BOOL` ?